doc/decisions/decision_process.md

```diff
@@ -64,6 +72,8 @@ We base our decision process and template on:
 Following subsections describe all steps a decision might run through.
 Only two of them are mandatory.
 
+In all steps we avoid discussions in issues or PRs but directly update the decision text with the different opinions and discuss the text of the decision PR.
```

Comment:
How can we update the decision text, if we do not have any discussion about it? This kinda looks like a "throwing stuff on the wall to see what sticks" approach from the author's side.